Automattic/wp-calypso
# Copy changes for performance plan

Fixes https://github.com/Automattic/wp-calypso/issues/74955

## Proposed Changes

* Copy updates for performance plan

Figma: lD0gTB4IqGsmSmsnjZNaP6-fi-2157-19583

For screenshots, please refer to issue https://github.com/Automattic/wp-calypso/issues/74955

## Testing Instructions

Please see https://github.com/Automattic/wp-calypso/issues/74955 for the changes in this PR. URL is included in the issue, review and make sure the screenshots with highlighted portion matches with this PR.

- Plans screen before purchase
- Checkout screen
- Plans upgraded screen
- Plans screen after purchase
- Free trial expiry screen


## Pre-merge Checklist
